powerlevel9k.zsh-theme

```diff
@@ -744,6 +744,38 @@ prompt_command_execution_time() {
   fi
 }
 
+################################################################
+# Determine the home folder unique path - this is needed for
+# truncate_to_unique to work when P9K_DIR_PATH_ABSOLUTE is not
+# set the true.
+#
+local home_path=""
+getUniqueHomeFolder() {
+  if [[ "$home_path" == "" ]]; then
+    local trunc_path directory test_dir test_dir_length
+    local -a matching
+    local -a paths
+    local cur_path='/'
+    # the first time we run the script, the working directory *should* be the home folder
```

Comment:
This comment worries me a little bit - can you expand on what you mean? If the segment doesn't get drawn for the first time from `/home/<user>`, will this code fail?

Reply:
I've tested it, even if the start folder isn't my home, it seems to work

![image](https://user-images.githubusercontent.com/13220636/42888190-fb273f7c-8aa7-11e8-82a6-edf686759de8.png)

Reply:
Usually, the terminal starts in the home folder, which is why I stated *should*, but if you check the next line, we test for the existence $HOME and use that, otherwise use ${PWD}.